app/frontend/src/features/messages/Messages.stories.tsx

```diff
@@ -1,10 +1,33 @@
+import * as React from "react";
 // also exported from '@storybook/react' if you can deal with breaking changes in 6.1
 import { Meta, Story } from "@storybook/react/types-6-0";
-import * as React from "react";
 import { Provider } from "react-redux";
 import { store } from "../../store";
 
-import Messages from "./Messages";
+import Messages from "./index";
+import MessageView, { MessageProps } from "./messagelist/Message";
+import { mockedService } from "../../__mocks__/service";
+
+Object.assign(mockedService, {
+  conversations: {
```

Comment:
Shouldn't we put this in the manual mock file instead?

Reply:
I do not know. Sometimes it is handy if you can make of some pre-mocked stuff. Sometimes it is handy to define your own.

Reply:
Is this a sane default to have? You can always override it with how you're doing it now right if it needs to be modified for specific cases?

Reply:
I put it there.